src/Utilities/UnitTests/ToolLocationHelper_Tests.cs

```diff
@@ -380,6 +380,12 @@ public void FindFrameworksPathRunningThisTest()
         [Fact]
         public void FindFrameworksPathRunningUnderWhidbey()
         {
+            if (!NativeMethodsShared.IsWindows)
```

Comment:
I think Roslyn does this with a test trait, which might be a better idea.  We should consider that in the future (not now)